src/app/pipes/nodedisplay.ts

```diff
@@ -50,7 +50,7 @@ export class NodeExternalUrlPipe implements PipeTransform {
                 return `https://bugzilla.redhat.com/show_bug.cgi?id=${node.id}`;
             case('distgitcommit'):
                 const baseUrl = 'http://pkgs.devel.redhat.com/cgit/';
-                if (node.repos.length) {
+                if (node.repos) {
```

Comment:
This change was necessary so that DistGitCommits would show up. However, I am unsure if it will cause problems elsewhere, although it did not break any tests.

Reply:
We use `node.repos.length` because an empty array evaluates to truthy.

You can see that with the following:
```javascript
> Boolean([])
true
```

How about changing this to?
```javascript
if (node.repos && node.repos.length) {
```

Reply:
On second thought, the frontend code is assuming `serialized_all` (which includes the relationships) at:
https://github.com/release-engineering/estuary-api/blob/master/estuary/utils/recents.py#L49

Changing that line should fix this issue.

Reply:
@sarah256 did you see this comment?

Reply:
@mprahl should I make a PR for this in the API before this PR gets merged? Also am not quite sure I understand what you want me to change in that line. Would it just be easier to do the if clause that you suggested?

Reply:
I'll try reexplain. The easy fix is to change the following line on the API:
https://github.com/release-engineering/estuary-api/blob/master/estuary/utils/recents.py#L49

To easy fix would be to change it from `serialized_node = node.serialized` to `serialized_node = node.serialized_all`. That will cause the JSON to included the direct relationships that nodes have. In the case of a dist-git commit, this is needed to know which repo the commit belongs to.